internal/grafanadi/service/podphase.go

```diff
@@ -85,6 +85,8 @@ func ConvertPodPhase2Frame(podPhases []*storagemodel.LifePhase) model.DataFrame
 				if msg != "" && msg != nil {
 					phaseMessage = msg.(string)
 				}
+				// 如果是eventObject则赋值message
```

Comment:
Please do not use Chinese characters in source files

Reply:
ok, i have deleted the Chinese characters